srcpkgs/xdg-desktop-portal-gnome/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'xdg-desktop-portal-gnome'
 pkgname=xdg-desktop-portal-gnome
 version=42.3
```

Comment:
xdg-desktop-portal-gnome 43.0 is avaliable. (will require xdg-desktop-portal 1.15)

Reply:
Can we update it separately or does it need gnome 43?

Reply:
Ah, you're right, my bad.